src/test/settings.test.ts

```diff
@@ -30,11 +30,10 @@ test('finbot > sets the appveyor token for a user', (t) => {
 
   respondStub.callsArgWith(1, response);
 
-  const robotBrain = new MockRobotBrain();
-  robot.brain = robotBrain;
-  const brainSetSpy = sinon.spy(robotBrain, 'set');
+  const secureBrain = new MockSecureBrain();

```

Comment:
[Use this to 'setup' the commonalities between these tests?](https://github.com/avajs/ava#before--after-hooks)